sources/zafira-services/src/main/java/com/qaprosoft/zafira/services/services/application/SettingsService.java

```diff
@@ -69,6 +74,14 @@ public Setting getSettingByType(SettingType type) {
         return settingsMapper.getSettingsByEncrypted(isEncrypted);
     }
 
+    @Transactional(readOnly = true)
+    public Map<Tool, Boolean> getToolsStatuses() {
+        Map<Tool, Boolean> toolSettings = new HashMap<>();
```

Comment:
Please rewrite so it will be more readable (start every other stream operation on a new line) and collect stream to hashMap vs manual hashMap creation and elements addition